website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
Looks like routing `Rule`s are [designed](https://github.com/CenterForOpenScience/osf.io/blob/cef7a1f89871bf624473b556ff70157b6701a1bf/framework/routing/__init__.py#L69-L70) to take a `dict` directly for cases such as ours here.